### ISSUE TITLE
Command-C shortcut for copying entries misbehaves on macOS

### ISSUE THREAD

**Issue by szhorvat** [JabRef/jabref#9169]
### JabRef version

```
JabRef 5.7--2022-08-05--73c111c
Mac OS X 10.14.6 x86_64 
Java 18.0.1 
JavaFX 18.0.1+2
```

### Operating system

macOS

### Checked with the latest development build

- [ ] I made a backup of my libraries before testing the latest development version.
- [ ] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Select an entry from the database. Choose one with an author name that does not start with C.
2. Press Command-C to copy it.

    Notice that "Copied 1 entry" will appear not once, but twice.

    The selection immediately jumps to the first entry having an author name starting with a C.

3. Paste the entry into another database.

    Notice that the entry that is pasted is not the one you originally selected, but the one that the selection jumped to.

In summary, it is not possible to copy an entry using keyboard shortcut, only using right-click menu.

**Comment by szhorvat** [comment]
May be the same as #9096

**Comment by dyannya** [comment]
Hi! I am interested on doing this, may I please get assigned to this?

**Comment by Siedlerchr** [comment]
I see the copied 1 entry still twice, but I can longer reproduce the other issue , the correct entry is now copied

**Comment by koppor** [comment]
This is JDK issue https://bugs.openjdk.org/browse/JDK-8088897.

**Comment by Siedlerchr** [comment]
I think I have found one additional culprit in our code as well. We don't filter out special keys. e.g. command, alt shift when jumping to the entry. This should not be the case
